src/mem/port.cc

```diff
@@ -103,54 +103,55 @@ class DefaultSlavePort : public SlavePort
     AddrRangeList getAddrRanges() const override { return AddrRangeList(); }
 };
 
-DefaultMasterPort defaultMasterPort;
```

Comment:
Are these variable names used anywhere in gem5?

Reply:
I searched the gem5 directory for other references and it appears to only be used in port.cc. During our discussion yesterday you renamed a variable and were able to see/rename all other references in the repo. I thought the option was "rename symbol" however when I select this option vscode never successfully finds the references it just has this loading message.
![image](https://user-images.githubusercontent.com/35640953/88855740-2beec280-d1a8-11ea-83b5-32433e17a6ea.png)